src/Form.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import { fetchData } from "./Api.jsx";
+import ResponseContent from "./ResponseContent.jsx";
+const Form = () => {
+  const [name, setName] = useState("");
+  const [password, setPassword] = useState("");
+  const [response, setResponse] = useState(null);
+  const [loading, setLoading] = useState(false);
+
+  const submitForm = async (e) => {
+    e.preventDefault();
+
+    setLoading(true);
+    const params = {
+      name,
+      password,
+    };
+    const answer = await fetchData(params);
+    setResponse(answer);
+    setLoading(false);
+  };
+
+  const hanndleNameFieldChange = (e) => {
+    e.preventDefault();
```

Comment:
Why is this here? `  e.preventDefault();`

Reply:
I've read that it's nice to have to prevent default behavior. But I guess I've misunderstood.